activerecord/lib/active_record/cache_key_with_version.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module ActiveRecord
+  module CacheKeyWithVersion
```

Comment:
I prefer to copy this method to relation than adding a new module just for this method.

Reply:
Reasonable, rebased accordingly.